x/programs/examples/imports/pstate/pstate.go

```diff
@@ -124,31 +128,31 @@ func (i *Import) getFn(caller *wasmtime.Caller, id int64, key int64) int64 {
 				zap.Error(err),
 			)
 		}
-		return -1
+		return nil, err
```

Comment:
bug: need to update the rust sdk to handle for errors now

Reply:
created an issue to hit up after this is merged. Temporarily returning -1 here
https://github.com/ava-labs/hypersdk/issues/666